eng/Versions.props

```diff
@@ -70,7 +70,7 @@
     <MicrosoftNETCoreILAsmVersion>6.0.0-rc.2.21451.6</MicrosoftNETCoreILAsmVersion>
     <MicrosoftNETSdkILVersion>6.0.0-rc.2.21451.6</MicrosoftNETSdkILVersion>
     <!-- Libraries dependencies -->
-    <SystemTextJsonVersion>6.0.0-rc.2.21451.6</SystemTextJsonVersion>
+    <SystemTextJsonVersion>6.0.6</SystemTextJsonVersion>
```

Comment:
Any reason no to take a dependency on a 7.0 version?  From a source-build perspective this would be better.

Reply:
I was just looking for a stable version, but I don't have any concerns trying out 7.0-rc1 for now and possibly the 7.0 RTM copy once we get ready to stabilize the releases library.

Reply:
I updated to 7.0.0-rc.1 for now and we can update again once we have 7.0.0 RTM